bulkresetcompletion.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Manually reset completion date for a course for a selected amount of users.
+ *
+ * @package     local_recompletion
+ * @copyright   2023 ChatGPT
```

Comment:
I'm not sure if this was intended as a joke, but I'd prefer to have your name here or just assign it to Catalyst IT.